addons/website/views/website_templates.xml

```diff
@@ -389,14 +389,107 @@
+                        <t t-call="portal.placeholder_user_sign_in">
+                            <t t-set="_link_class" t-value="'nav-link'"/>
+                        </t>
+                    </li>
+                    <!-- User Dropdown -->
+                    <li t-ignore="true" t-if="website.user_id != user_id" class="dropdown nav-item ml-4">
+                        <t t-call="portal.user_dropdown">
+                            <t t-set="_icon" t-value="true"/>
+                            <t t-set="_toggle_class" t-value="'nav-link'"/>
+                            <t t-set="_dropdown_menu_class" t-value="'position-absolute dropdown-menu-right'"/>
+                        </t>
+                    </li>
+                </ul>
+                <!-- Navbar Toggler -->
+                <t t-call="website.navbar_toggler">
+                    <t t-set="_toggler_class" t-value="'ml-4'"/>
+                </t>
+                <!-- Button -->
+                <t t-call="website.placeholder_header_button">
+                    <t t-set="_btn_class" t-value="'btn btn-primary ml-4'"/>
+                </t>
+                <!-- Navbar Collapse -->
+                <div id="top_menu_collapse" class="collapse navbar-collapse">
+                    <!-- Separator -->
+                    <div id="oe_structure_header_hamburger_1" class="oe_structure">
+                        <div class="s_hr text-left pt16 pb16">
+                            <hr class="w-100 mx-auto" style="border-top-width: 1px; border-top-style: solid; border-color: var(--300);"/>
+                        </div>
+                    </div>
+                    <t t-call="website.navbar_nav">
+                        <!-- Menu -->
+                        <t t-foreach="website.menu_id.child_id" t-as="submenu">
+                            <t t-call="website.submenu">
+                                <t t-set="item_class" t-value="'nav-item'"/>
+                                <t t-set="link_class" t-value="'nav-link'"/>
+                            </t>
+                        </t>
+                        <!-- Cart -->
+                        <t t-call="website.placeholder_header_cart">
+                            <t t-set="_text" t-value="true"/>
+                            <t t-set="_item_class" t-value="'nav-item'"/>
+                            <t t-set="_link_class" t-value="'nav-link'"/>
+                        </t>
+                        <!-- Wishlist -->
+                        <t t-call="website.placeholder_header_wishlist">
+                            <t t-set="_text" t-value="true"/>
+                            <t t-set="_item_class" t-value="'nav-item'"/>
+                            <t t-set="_link_class" t-value="'nav-link'"/>
+                        </t>
+                    </t>
+                    <div id="oe_structure_header_hamburger_2" class="oe_structure">
+                        <div class="row align-items-center">
```

Comment:
I did not explain that correctly.

For an oe_structure to be properly customized, we need:
- A div with the "oe_structure" class
- An ID starting by "oe_structure_"
- The div needs to be empty

To add default content to the div, you have to simulate the user already edited it by making a xpath filling it. There is a unique exemple in Odoo codebase, do exactly the same thing:
```xml
<xpath expr="//*[hasclass('oe_structure')][@id='oe_structure_forum_all_top']" position="replace">
```

We obviously have to improve this systeme in the future ;)

Reply:
@Cocographique If it is still not working after that, try to wrap the original oe_structure in a simple <div> and I think it will work